apps/react-app/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "vite --port 8080 --open"
+    "start": "vite --port 8080 --open",
+    "build": "vite build"
```

Comment:
Why do you need the build command on these apps?

Reply:
After talk with @brauliodiez, these apps will be deployed in a web server so we alse will need to build them. Thanks

Reply:
Maybe for publish packages, I think we should update the `publish-packages` command to: `"publish-packages": "turbo run build --filter=./packages/* && changeset publish"`

Reply:
I have changed the publish-packages command as you suggested